include/hashinator/hashinator.h

```diff
@@ -1214,6 +1246,24 @@ class Hashmap {
       }
       return elements.size();
    }
+   
```

Comment:
Are the old versions of `extractKeysByPattern` on lines 1233 and 1267 still needed now that the KeyAlloc versions exist?

Reply:
Nope those methods should stay in my opinion. They are currently used in the unit tests. Now the new  methods will only be called if the allocator is provided in the template argument list